<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="dialogButtons">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>480</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="glLayout"/>
  </widget>
  <widget class="QDial" name="dialRotx">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RotX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>RotY</string>
   </property>
  </widget>
  <widget class="QDial" name="dialRoty">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>RotZ</string>
   </property>
  </widget>
  <widget class="QDial" name="dialRotz">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>30</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbx">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sby">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>110</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbz">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>110</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbSelectSurface1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface1</string>
   </property>
   <attribute name="buttonGroup">
    <string>bgSurfaceSelect</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rbSelectSurface2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface2</string>
   </property>
   <attribute name="buttonGroup">
    <string>bgSurfaceSelect</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rbSelectSurface3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>220</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface3</string>
   </property>
   <attribute name="buttonGroup">
    <string>bgSurfaceSelect</string>
   </attribute>
  </widget>
  <widget class="QDial" name="dialTheta">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>180</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dialPhi">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>180</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Theta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>160</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bbDoTest">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>290</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Do test</string>
   </property>
  </widget>
  <widget class="QDial" name="dialShiftX">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>280</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDial" name="dialShiftY">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>280</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>280</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>280</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbWireSurface">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>260</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>height:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>440</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shiftXVal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>350</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shiftYVal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>350</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bbShowSlice">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>380</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Slice</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>380</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sliceStepVal">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>380</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="thetaVal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>250</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phiVal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>250</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>410</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>deg step:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="degStepVal">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>410</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bbPolarSlice">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Polar Slice</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>MainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bgSurfaceSelect"/>
 </buttongroups>
</ui>
